modules/ROOT/pages/migration/upgrading_5.0.x_7.0.0.adoc

```diff
@@ -0,0 +1,169 @@
+
+IMPORTANT: Check below, if you are affected by breaking changes and prepare all steps mentioned before you start the upgrade.
+
+== Upgrade Steps
+
+. Shut down the Infinite Scale instance and +
+*do not start it after downloading the binary or image*!
+. Download and install Infinite Scale
+. Manage Breaking Changes
+. Start Infinite Scale
+
+== Download and Install Infinite Scale
+
+Download and install Infinite Scale via:
+
+* *Binary* +
+Follow the xref:depl-examples/minimal-bare-metal.adoc#installation[Installation] section of the bare metal deployment example.
+
+* *Docker and Docker Compose*
++
+[source,bash,subs="attributes+"]
+----
+docker pull owncloud/ocis:7.0.0
+----
+
+** When using `docker run`
+
+*** Update the image version used in the docker run command.
+
+** When using `docker compose`
+
+*** Update _every_ compose file where the `ocis image` is referenced accordingly.
+
+*** If you have used the deployment examples based on _rolling releases version 6.x_ either for xref:depl-examples/ubuntu-compose/ubuntu-compose-prod.adoc[Local Production Setup] or xref:depl-examples/ubuntu-compose/ubuntu-compose-hetzner.adoc[Deployment on Hetzner], read the *Updating* section of that pages carefully. 
+
+== Manage Breaking Changes
+
+* All environment variables that were marked for deprecation in Infinite Scale release 5.0.x have finally been removed.
+* A new mandatory Infinite Scale config setting for the `activitylog` service named `service_account` has been added.
+
+=== How to Identify if You Are Affected
+
+* If you are using deprecated environment variables in your config.
+** See the xref:deployment/services/env-var-changes.adoc[Added and Removed Environment Variables] documentation.
+
+* Because the `activitylog` service has been added, everyone who upgrades is affected.
+
+In all other cases, there is nothing that needs to be done.
+
+=== How to Manage the Change
+
+* Manage deprecated variables
+** Environment variables that have been deprecated without successor can safely be removed from the configuration. These envvars do not harm as they are not used anymore.
+** Environment variables that have been deprecated with a successor need to be updated accordingly.
+
+* Add new Infnite Scale config settings. +
+To do so, the following steps need to be taken, here is the overview:
++
+--
+. Get the new binary/image, but do not start Infinite Scale
+. Get the config diff via command
+. Apply the diff created
+--
++
+See the detailed steps below.
+
+* You can now start Infinite Scale as usual
+
+{empty}
+
+Update Config Settings::
+* If not already done, get the new Infinite Scale version as described in xref:download-and-install-infinite-scale[Download and Install Infinite Scale].
+
+* Run the `ocis init --diff` command: +
+Because this will now run the new ocis code, it will create in case of changes found two files:
```

Comment:
```suggestion
Because this will now run the new ocis code, it will create two files to help manage the needed changes:
```

(what happens if no changes are needed? are no files created? or is `ocis.config.patch` and empty file?)